candidate-standard/openapi/EDR_OpenAPI.json

```diff
@@ -1,9 +1,9 @@
 {
   "openapi": "3.0.2",
   "info": {
-    "title": "Environmental Data Retreival API's",
-    "description": "Example of Candidate Environmental Data Retreival API OpenAPI docs",
-    "version": "0.0.5"
+    "title": "Environmental Data Retrieval API's",
+    "description": "Example of Candidate Environmental Data Retrieval API OpenAPI docs",
+    "version": "0.0.6"
```

Comment:
Should the version here be one ahead of the version of the spec? in: candidate-standard/EDR-candidate-specification.adoc ?

Reply:
@dblodgett-usgs No the two version values should be synchronised

Reply:
👍